### PR TITLE
add protocol to iCal link and reword language

### LINKED ISSUE #185: Add to other calendar should have protocol
Currently the URL is
`calendar.google.com/calendar/ical/c_1pcp6odi9qfe276tpuob8h00ms@group.calendar.google.com/public/basic.ics` 
but it should be
`https://calendar.google.com/calendar/ical/c_1pcp6odi9qfe276tpuob8h00ms@group.calendar.google.com/public/basic.ics`

Note that the webcal link should still be
`webcal://calendar.google.com/calendar/ical/c_1pcp6odi9qfe276tpuob8h00ms@group.calendar.google.com/public/basic.ics`
not
`webcal://https://calendar.google.com/calendar/ical/c_1pcp6odi9qfe276tpuob8h00ms@group.calendar.google.com/public/basic.ics`

### PR DESCRIPTION
**Pull request by zalhabash**
Fixes #185 
- Add protocol to iCal link b/c we mistakenly left it out
- Reword language in "Add to Other Calendar button" description to "subscribe". Makes it easier to find the place to paste the iCal link into other calendar apps.

### CODE REVIEW

**Review by nstringham — DISMISSED** [review]

File: src/lib/FullCalendar.svelte [hunk 1]
```diff
@@ -123,12 +123,9 @@
 >
 	<article>
 		<h2>Add to Your Calendar</h2>
-		<p>
-			Copy the iCal url into your calendar app to automatically add Kendo Club events to your
```
Comment (nstringham) [inline src/lib/FullCalendar.svelte:127]: We might still want to let them know that they need to copy it.  What about "Copy this iCal url into your calendar app to subscribe to the kendo at umich calendar" I don't know.

Comment (zalhabash) [reply src/lib/FullCalendar.svelte:127]: That's really good, yeah!

File: src/lib/FullCalendar.svelte [hunk 2]
```diff
@@ -124,11 +124,10 @@
 	<article>
 		<h2>Add to Your Calendar</h2>
 		<p>
-			Copy the iCal url into your calendar app to automatically add Kendo Club events to your
-			calendar.
+			Copy this iCal link into your calendar app to subscribe to the Kendo Club at Umich calendar.
```
Comment (nstringham) [inline src/lib/FullCalendar.svelte:127]: Did you change URL to link intentionally?

Comment (zalhabash) [reply src/lib/FullCalendar.svelte:127]: No I don't think so

Comment (nstringham) [reply src/lib/FullCalendar.svelte:127]: Would you like to change it back?

Comment (zalhabash) [reply src/lib/FullCalendar.svelte]: Sorry, wasn't able to at the time. Just did so.

File: src/lib/FullCalendar.svelte [hunk 3]
```diff
@@ -124,11 +124,10 @@
 	<article>
 		<h2>Add to Your Calendar</h2>
 		<p>
-			Copy the iCal url into your calendar app to automatically add Kendo Club events to your
-			calendar.
+			Copy this iCal url into your calendar app to subscribe to the Kendo Club at Umich calendar.
 		</p>
 		<div role="group">
-			<input value={icalUrl} readonly />
+			<input value={'https://' + icalUrl} readonly />
```
Comment (nstringham) [inline src/lib/FullCalendar.svelte:130]: I usually prefer to do `https://{icalUrl}` but I guess I this is fine.

**Review by nstringham — APPROVED** [review]

File: src/lib/FullCalendar.svelte (hunk 3 above)
Comment (zalhabash) [reply src/lib/FullCalendar.svelte:130]: I'm not completely sure how to do that since this is inside of a script tag

Comment (nstringham) [reply src/lib/FullCalendar.svelte:130]: It's not inside a script tag

Comment (zalhabash) [reply src/lib/FullCalendar.svelte:130]: Oh you're right, oops

**Pull request merged**